src/authorizer.js

```diff
@@ -94,21 +94,22 @@ class Authorizer {
   async getPolicy(request) {
     this.logFunction({ level: 'INFO', title: 'Authorizer.getPolicy()', data: request });
```

Comment:
Now we have logging of the `path` twice during execution. What if instead of logging this line here, we log the full `request` in each of these WARNs.  If we still want to keep this log line we could move it down to L124

Reply:
I made some changes to address this, let me know what you think. I figure it made sense to change the log message, but we could also keep it the same.